exercises/diffie-hellman/example.js

```diff
@@ -0,0 +1,48 @@
+
+var DiffieHellman = function(p, g){
+  this.p = p;
+  this.g = g;
+
+  // array of first 1000 primes.
+  this.PRIMES= [
```

Comment:
My feelings are a bit stronger with `PRIMES` than with `p` and `g`.

I would declare `PRIMES` outside of `DiffieHellman`. As long as it's not exported, it won't be visible out of the module.